x-pack/plugins/siem/public/components/charts/common.tsx

```diff
@@ -25,27 +32,21 @@ export const ChartHolder = () => (
   </FlexGroup>
 );
 
-export interface AreaChartData {
-  key: string;
-  value: ChartData[] | [] | null;
-  color?: string | undefined;
-}
-
 export interface ChartData {
   x: number | string | null;
   y: number | string | null;
   y0?: number;
   g?: number | string;
 }
 
-export interface BarChartData {
+export interface ChartConfigsData {
```

Comment:
In recent case, I had two different types of chart data to highlight the difference between `areaChartData` and `barChartData`:
- In areaChart, each chartData represent the `same data type` over `different time periods`.
- In barChart, as we shows only `single value` for a data type, so limited the element count to allow only one element inside the array.
After having a chat with @XavierM , I think it would make more sense to remove the limitation for numbers of data inside BarChartData to make it more reusable. Therefor, there will be no difference between the form of BarChartData and AreaChartData, they are going to share the same type `ChartConfigsData`.